sklearn_to_astartes.md

```diff
@@ -79,7 +79,7 @@ X_train, X_test, y_train, y_test = train_test_split(
 ## Step 5. Useful `astartes` Features
 
 ### `return_indices`: Improve Code Clarity
-When providing `X`, `y`, and `labels` it can become cumbersome to unpack all of arrays, and there are cirumstances where the indices of the train/test data can be useful (for example, if `y` or `labels` are large, memory-intense objects). By default, `astartes` will return the arrays themselves, but it can also return just the indices for the user to manipulate according to their needs:
+There are circumstances where the indices of the train/test data can be useful (for example, if `y` or `labels` are large, memory-intense objects), and there is no way to directly return these indices in `sklearn`. `astartes` will return the sampling splits themselves by default, but it can also return the indices for the user to manipulate according to their needs:
```

Comment:
I may not be thinking of all scenarios, but most likely X will be a larger object than y or labels. Also, I think the way to obtain indices from sklearn is to just pass in np.arange() which I believe is what the random sampler does. However, I understand what you mean so I don't think it's worth updating the markdown description.

Reply:
Agreed that `X` should likely be larger - I recall this being an issue with random sampling, but thought it might extend here. Certainly individually each row of `X` would be larger than the corresponding target or label, but when there are huge numbers of samples it still might provide speedup.

I also recall passing in ranges to `train_test_split` being an effective workaround, but I don't want to point people in that direction now that we have a much more compelling option ;)